contrib/sarplus/scala/compat/src/main/scala/com/microsoft/sarplus/compat/spark/since3p2defvisible.scala

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.sarplus.spark
```

Comment:
all the scala files should also include the license header

Reply:
Added